server/controllers/api/users/index.ts

```diff
@@ -188,6 +188,7 @@ async function createUser (req: express.Request, res: express.Response) {
     role: body.role,
     videoQuota: body.videoQuota,
     videoQuotaDaily: body.videoQuotaDaily,
+    videosHistoryEnabled: CONFIG.USER.HISTORY.ENABLED_BY_DEFAULT,
```

Comment:
Also add it in https://github.com/Chocobozzz/PeerTube/blob/develop/server/lib/auth/oauth-model.ts#L224